python/doc/examples/functional_modeling/vectorial_functions/plot_createMultivariateFunction.py

```diff
@@ -0,0 +1,168 @@
+enumerateFunction = ot.LinearEnumerateFunction(len(familyColl))
+
+familyFunction = ot.TensorizedUniVariateFunctionFactory(familyColl, enumerateFunction)
+k = 3
+f = familyFunction.build(k)
+print('input dimension = ', f.getInputDimension())
+print('output dimension = ', f.getOutputDimension())
+
+# %%
+# If we want to use an orthogonal univariate polynomials family, then we have to cast the family in the  :class:`~openturns.OrthogonalUniVariatePolynomialFunctionFactory` class.
+
+# %%
+# For example, we use the  Jacobi orthogonal univariate polynomials family.
+
+univFuncFamily_Jacobi = ot.OrthogonalUniVariatePolynomialFunctionFactory(ot.JacobiFactory())
+
+
+# %%
+# Case 3:  we use a tensorization of an orthogonal univariate polynomials family
+# ==============================================================================
+#
+# In that case, the univariate polynomials are orthogonal with respect to a measure :math:`\mu`.
+#
+# We have the family of polynomials: :math:`(x_i \mapsto \phi^i_k(x_i))_{k \geq 0}`
+# for :math:`1 \leq i \leq d`. The :meth:`enumerate` function stores of the :class:`~openturns.LinearEnumerateFunction` class the way
+# the multivariate basis is enumerated: *enumerate(n)* is a multi-index
+# :math:`(\ell_1, \dots, \ell_d)` such that the function number :math:`n`
+# is
+#
+# .. math::
+#     f_n(\vect{x}) = \prod_{k=1}^d \phi^i_{\ell_i}(x_i).
+#
+# We use the  :class:`~openturns.OrthogonalProductPolynomialFactory` class.
+
+# %%
+# For example, we consider : :math:`f: \mathbb{R}^2 \mapsto \mathbb{R}`, and:
+#
+# - dimension of :math:`x_1`: the family of Jacobi polynomials: :math:`(x_1 \mapsto x_1^k){k \geq 0}`,
+# - dimension of :math:`x_2`: the family of Hermite polynomials: :math:`(x_2 \mapsto \phi^2_k(x_2))_{k \geq 0}`.
+#
+
+univFuncFamily_Jacobi = ot.JacobiFactory()
+univFuncFamily_Hermite = ot.HermiteFactory()
+familyColl = [univFuncFamily_Jacobi, univFuncFamily_Hermite]
+enumerateFunction = ot.LinearEnumerateFunction(len(familyColl))
+familyFunction = ot.OrthogonalProductPolynomialFactory(familyColl, enumerateFunction)
+f = familyFunction.build(3)
+print('input dimension = ', f.getInputDimension())
+print('output dimension = ', f.getOutputDimension())
+
+# %%
+# We get the measure:
+measure_Jacobi = ot.JacobiFactory().getMeasure()
+measure_Hermite = ot.HermiteFactory().getMeasure()
+
+
+# %%
+# Case 4: we use a tensorization of an orthogonal univariate functions family
+# ===========================================================================
+#
+# In that case, the univariate functions are orthogonal with respect to a measure :math:`\mu`.
+#
+# We have the family of functions: :math:`(x_i \mapsto \phi^i_k(x_i))_{k \geq 0}`
+# for :math:`1 \leq i \leq d`.. The :math:`enumerate` function of the :class:`~openturns.LinearEnumerateFunction` class stores the way
+# the multivariate basis is enumerated: *enumerate(n)* is a multi-index
+# :math:`(\ell_1, \dots, \ell_d)` such that the function number *n* is
```

Comment:
The equation is wrong, because the index of the product is $k$, while the index of the multi-index is $i$:

![image](https://user-images.githubusercontent.com/31351465/235374500-27c1b63a-7634-4a42-8f6d-70b3b2266301.png)

Hence, the product is performed with a constant multi-index $\ell_i$, which is not your goal I presume. Moreover, the standard notation for multi-indices is $\boldsymbol{\alpha}$, as already mention in the documentation of the [EnumerateFunction](http://openturns.github.io/openturns/latest/user_manual/_generated/openturns.EnumerateFunction.html).

```suggestion
# :math:`\vect{\alpha} = (\alpha_1, \dots, \alpha_d)` such that the function number *n* is
```

Reply:
ok